### PR TITLE
[FIX] web: prevent URL click from opening a record

### PR DESCRIPTION
**Pull request by seb-odoo**
If we click on a link, we only want to open the URL, and not the record when the button is on a list.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by seb-odoo** [comment]
@aab-odoo could you have a look please?

I'm not sure how to write a test to check that a link is open in the browser and that nothing else happens.

**Comment by seb-odoo** [comment]
@aab-odoo thanks for the feedback, I made the changes and added a test.

Can you have a look again please?

File: addons/web/static/tests/fields/relational_fields_tests.js [hunk 1]
```diff
@@ -9665,6 +9665,37 @@ QUnit.module('relational_fields', {
...
+            View: FormView,
+            model: 'partner',
+            data: this.data,
+            arch:'<form string="Partners">' +
+                    '<field name="turtles">' +
+                        '<tree>' +
+                            '<field name="display_name" widget="email"/>' +
+                            '<field name="turtle_foo" widget="url"/>' +
+                        '</tree>' +
+                        '<form></form>' +
```
Comment (aab-odoo) [inline addons/web/static/tests/fields/relational_fields_tests.js]: no need

**Comment by seb-odoo** [comment]
Yes, if I don't add the empty <form> the tests just crash before the fix because it tries to open it but it doesn't exist. And by adding the empty <form> the tests fail because length is 1.

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**